src/main/kotlin/org/jetbrains/research/testspark/collectors/CollectorsHelper.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.research.testspark.collectors
+
+class CollectorsHelper {
```

Comment:
Can you please add comments and explain what is this class and what are the regex patterns introduced here?